test/scalability/default_rangespec.yaml

```diff
@@ -1,15 +1,32 @@
-# Until we have a clear picture on how the setup
-# performs in CI keep the values "very relaxed"
+# The values are computed based on the result of 5 trial runs:
+# - #1782760671465705472
+# - #1782764439129296896
+# - #1782768037514973184
+# - #1782772615836864512
+# - #1782775995984515072
 cmd:
-  maxWallMs: 3600_000 #1h 
-  maxUserMs: 3600_000
-  maxSysMs: 3600_000
-  maxrss: 1024_000 #1000MiB
+  # Average value 351116.4 (+/- 0.9%), setting at +5%
+  maxWallMs: 368672
```

Comment:
Nit:
I would round up to the next second or even next minute.

Reply:
done